### DISCUSSION TITLE
How to effectively utilize the high command mode

### DISCUSSION THREAD

**Discussion by MarkTwoFive** [official-antistasi-community/A3-Antistasi#3387] [Q&A] [answered]
Hey 👋🏼 ,
I've been trying to be a more effective commander to my team but I'm having some issues using the high command mode to get the AI squads to actually do something.  

Switching to HC using the keybind and using the interaction menu to set waypoints for the AI doesn't seem to do anything.  
The only thing working for me so far is fast travel so at least I can move them around friendly locations which helps a bit.

Of course I can also take control of their squad leader to give them orders "the vanilla way" but that is inconvenient because it doesn't include the AI squad leader and especially because it causes the AI replacing my player unit to issue useless commands to my squad.  

Do you have any tips/tricks for me on how to use the HC mode to our advantage?

**Comment by jaj22** [comment, marked as answer]
Normally they move to the waypoints you give them. It's mostly vanilla functionality. Strip down your modset and see if it works then.

**Reply by MarkTwoFive** [reply]
Thanks for the quick response.
I've started going through the mods and it appears you're exactly right. I'm still trying to pinpoint the exact mod though. It seems it is not as obvious as just "whatever mod does something to the map screen".
